js/confidenceSliderView.js

```diff
@@ -94,7 +94,12 @@ export default class ConfidenceSliderView extends SliderView {
   }
 
   onLinkedSubmittedChanged(linkedModel) {
+    this.$('.component__body-inner').html(linkedModel.get('body'));
```

Comment:
```suggestion
    this.$('.component__body-inner').html(this.model.get('body'));
```
This should probably be the body from the model for this instance and not the linked instance.